packages/yoroi-extension/app/containers/wallet/voting/VotingPage.js

```diff
@@ -102,24 +135,93 @@ export default class VotingPage extends Component<Props> {
         </VerticallyCenteredLayout>
       );
     }
-
     // keep enabled on the testnet
+    const catalystRoundInfo = this.generated.stores.substores.ada.votingStore.catalystRoundInfo;
     if (!environment.isTest()) {
-      const isLate = new Date() >= roundInfo.endDate;
-      const isEarly = new Date() <= roundInfo.startDate;
-      if (
-        selected.getParent().getNetworkInfo().NetworkId === networks.CardanoMainnet.NetworkId &&
-        (isEarly || isLate)
-      ) {
+      if (!catalystRoundInfo || (!catalystRoundInfo.currentFund && !catalystRoundInfo.nextFund)){
         return (
-          <RegistrationOver roundNumber={
-              isLate
-                ? roundInfo.nextRound
-                : roundInfo.nextRound - 1
-            }
+          <RegistrationOver
+            title={intl.formatMessage(messages.unavailableTitle)}
+            subtitle={intl.formatMessage(messages.unavailableSubtitle)}
           />
-        );
-      }
+          )
```

Comment:
Please fix the indentation and end the statement with a semicolon. Also apply to the below lines.

Reply:
Okay